test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -14647,6 +14647,11 @@ def __init__(self, job, cmd, host):
         self.cmd = cmd
         self.jobid = None
         self.hostname = host
+        self.runas_user = ""
```

Comment:
I think you can get rid of this instance variable, and use local scope variable in below `if shasta` block ([here](https://github.com/openpbs/openpbs/pull/1950/files#diff-fef109b45f82b3a763d1bc74763fb067R14704))